python/dolfinx/mesh.py

```diff
@@ -218,3 +221,166 @@ def MeshTags(mesh: Mesh, dim: int, indices: np.ndarray, values: np.ndarray) -> t
 
     fn = _meshtags_types[dtype]
     return fn(mesh, dim, indices.astype(np.int32), values)
+
+
+def create_interval_mesh(comm: _MPI.Comm, nx: int, points: list, ghost_mode=GhostMode.shared_facet,
```

Comment:
No need for 'mesh' in the name when in the mesh module. See https://gist.github.com/sloria/7001839.